tests/specs/getLocalChanges.js

```diff
@@ -40,46 +38,43 @@ test('getLocalChanges()', function (t) {
   .then(function () {
     return api.getLocalChanges()
   })
-  .then(function (arr) {
-    t.equal(arr[0] > arr[1], true, 'local changes detected')
+  .then(function (localReplicationDoc) {
+    t.equal(localReplicationDoc.length, 3, 'local changes detected, 3 docs')
```

Comment:
yeah, in this test I'd leave the `foo` properties. And then do the `[changedDocs[0].foo, changedDocs[1].foo, changedDocs[2].foo].sort()` sort thing like we did else where